financialaid/views_test.py

```diff
@@ -1004,7 +910,22 @@ class CoursePriceListViewTests(FinancialAidBaseTestCase, APIClient):
     @classmethod
     def setUpTestData(cls):
         super().setUpTestData()
-        cls.course_price_url = reverse("course_price_list")
+        cls.username = "{}_edx".format(cls.profile.user.username)
+        cls.staff_username = "{}_edx".format(cls.staff_user_profile.user.username)
+        cls.instructor_username = "{}_edx".format(cls.instructor_user_profile.user.username)
+        cls.profile.user.social_auth.create(
+            provider=EdxOrgOAuth2.name,
+            uid=cls.username,
+            extra_data={"access_token": "fooooootoken"}
+        )
+        cls.staff_user_profile.user.social_auth.create(
+            provider=EdxOrgOAuth2.name,
+            uid=cls.staff_username,
+            extra_data={"access_token": "fooooootoken"}
+        )
+
+        cls.course_price_url = reverse("course_price_list", args=[cls.username])
+        cls.staff_course_price_url = reverse("course_price_list", args=[cls.staff_username])
```

Comment:
do you mean just to check that the user doesn't see course prices for a program they're not enrolled in?